package-examples/subpackages/wordpress/README.md

```diff
@@ -0,0 +1,92 @@
+wordpress
+==================================================
+
+# NAME
+
+  wordpress
+
+# SYNOPSIS
+  
+  This is an example of a kpt package which has a subpackage in it. 
+  Here are the steps to get, view, set and apply the package contents
+  
+  ### Fetch a remote package
+  Get the example package on to local using `kpt pkg get`
+  
+  $ kpt pkg get https://github.com/GoogleContainerTools/kpt.git/package-examples/subpackages/wordpress \
+    wordpress
+  
+      fetching package /package-examples/subpackages/wordpress from https://github.com/GoogleContainerTools/kpt to wordpress
+  
+  ### View the package contents
+  List the package contents in a tree structure.
+  
+  $ kpt cfg tree wordpress/
+  
+      wordpress
+      ├── [Kptfile]  Kptfile wordpress
+      ├── [wordpress-deployment.yaml]  Deployment wordpress
+      ├── [wordpress-deployment.yaml]  Service wordpress
+      ├── [wordpress-deployment.yaml]  PersistentVolumeClaim wp-pv-claim
+      └── Pkg: mysql
+          ├── [Kptfile]  Kptfile mysql
+          ├── [mysql-deployment.yaml]  PersistentVolumeClaim mysql-pv-claim
+          ├── [mysql-deployment.yaml]  Deployment wordpress-mysql
+          └── [mysql-deployment.yaml]  Service wordpress-mysql
+  
+  The fetched package contains [setters]. Invoke [list-setters] command to list
+  the [setters] recursively in all the packages.
+  
+  $ kpt cfg list-setters wordpress/
+  
+      wordpress/
+               NAME             VALUE      SET BY   DESCRIPTION   COUNT   REQUIRED  
+        gcloud.core.project   PROJECT_ID                          3       No        
+        image                 wordpress                           1       No        
+        tag                   4.8                                 1       No        
+        teamname              YOURTEAM                            3       Yes       
+      
+      wordpress/mysql/
+               NAME             VALUE      SET BY   DESCRIPTION   COUNT   REQUIRED  
+        gcloud.core.project   PROJECT_ID                          3       No        
+        image                 mysql                               1       No        
+        tag                   5.6                                 1       No        
+        teamname              YOURTEAM                            3       Yes       
+        
+   You may notice that the [auto-setter] `gcloud.core.project` is already set if you
+   have `gcloud` configured on your local.
+  
+  ### Provide the setter values
+  [Set] all the [required setters]. You may set other setters, either selectively
+  in few of the packages or in all the them. `--recurse-subpackages(-R)` flag is
+  `default:false` for set command. If not invoked, the set operation is performed
+  only on the resource files of parent package and not the subpackages.
```

Comment:
Rephrase:

"By default, the 'set' command is performed
  only on the resource files of parent package and not the subpackages. This behavior may be changed by setting  `--recurse-subpackages(-R)`..."

General comments
- First describe the behavior. How that behavior is changed is a detail that follows.
- Be consistent between 'operation', 'action', and 'command'. It's used interchangeably and is confusing. In this context, CLIs have 'commands'.

Apply these consistency everywhere.